addons/Popochiu/Editor/MainDock/TabRoom.gd

```diff
@@ -60,10 +69,36 @@ onready var _btn_resource_script: Button = _tool_buttons.get_node('BtnResourceSc
 
 # ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ GODOT ░░░░
 func _ready() -> void:
+	$PopochiuFilter.groups = _types
+	
+	# Setup the button that will allow to add characters to the room
```

Comment:
Out of curiosity, why have you got a tendency to create UI elements programmatically instead of having them in scenes?

I took another road with the inspector plugin for the importer to avoid cluttering the code with those declarations, but maybe I'm missing something?